src/elbo.jl

```diff
@@ -0,0 +1,244 @@
+"Likelihoods which take a scalar (or vector of scalars) as input and return a single scalar."
```

Comment:
what would be the "vector of scalars" case ?

Reply:
I'm not sure what I meant by that....